client/src/redux/sagas/datafiles.sagas.js

```diff
@@ -596,3 +597,178 @@ export function* trashFile(system, path, id) {
     });
   }
 }
+
+export const getLatestApp = async name => {
```

Comment:
Do we need unit tests for this? or maybe a follow up ticket?

Reply:
I think maybe a follow up ticket

Reply:
cool.